src/Modifiers/CoreModifiers.php

```diff
@@ -2232,6 +2232,66 @@ public function stripTags($value, $params, $context)
         return Str::stripTags($value, (array) $tags);
     }
 
+    /**
+     * Make str_pad() with padding available as a modifier.
+     *
+     * Example: {{ my_index | str_pad_both:2:0:left }}
+     *
+     * @param  string  $value  The value to be modified.
+     * @param  array  $params  Any parameters used in the modifier.
+     * @return string
+     */
+    public function strPad(string $value, array $params): string
+    {
+        $pad_length = Arr::get($params, 0);
+        $pad_string = Arr::get($params, 1, ' ');
+        $pad_type = constant('STR_PAD_'.Str::upper(Arr::get($params, 2, 'RIGHT')));
```

Comment:
I had no idea `constant()` was even a thing. Neat!

Reply:
Me neither, until I stumbled upon it, but it's been there since PHP 4 :D